setup.py

```diff
@@ -29,14 +29,15 @@
     install_requires=[
         "aiohttp",
         "ddsketch",
+        "ddtrace",
         "msgpack",
+        "pathlib",
```

Comment:
why do we need pathlib? The minimum required python version includes pathlib in the stdlib?

Reply:
didn't know that, removed it